app/util/autoversioning.py

```diff
@@ -51,7 +51,7 @@ def _calculate_source_version():
         hash_extension = '' if head_commit_is_on_trunk else '-{}'.format(head_commit_hash[:7])
         mod_extension = '' if not _repo_has_uncommited_changes() else '-mod'
         return '{}.{}{}{}'.format(_MAJOR_MINOR_VERSION, commit_count, hash_extension, mod_extension)
-    except subprocess.CalledProcessError:
+    except (subprocess.CalledProcessError, FileNotFoundError):
```

Comment:
Haha, fun. This is a good safeguard for packaging. Is that what the assert in setup.py guards against?

Now that I'm thinking about it, I wonder at what point the clusterrunner service would fail if git was missing. Probably not on startup. 😬

Reply:
Yeah, I hit this bug and the rpm was building with version 0.0. I made a change however to remove the assert in setup.py